test/src/module/common/ioTlsTest.c

```diff
@@ -949,5 +950,52 @@ testRun(void)
         TEST_RESULT_PTR_NE(statToJson(), NULL, "check");
     }
 
+    // *****************************************************************************************************************************
+    if (testBegin("harnessIo"))
+    {
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("shimmed ioSessionFd() returns defined arbitrary file descriptor");
+
+        struct addrinfo hints = (struct addrinfo)
+        {
+            .ai_family = AF_UNSPEC,
+            .ai_socktype = SOCK_STREAM,
+            .ai_protocol = IPPROTO_TCP,
+        };
+
+        const char *port = "7777";
+        const char *hostBad = "172.31.255.255";
```

Comment:
We seem to be defining these over and over.. just wonder if maybe we should put them in one place?  Probably a question for @dwsteele really.

Reply:
Perhaps, but this test in general seems to be overkill. Seems like it would be better to just pass a bogus fd (e.g. HRNIO_FILE_DESCRIPTOR - 1) to `sckSessionNew()` and test to be sure the shim works that way.

Honestly not sure we need this test at all, since it should be pretty obvious in the test module if it is not working.